app/models/event.rb

```diff
@@ -11,6 +11,7 @@ class Event < ApplicationRecord
   has_many :session_formats, dependent: :destroy
   has_many :taggings, through: :proposals
   has_many :ratings, through: :proposals
+  has_many :sponsors
```

Comment:
I can see why you would choose to connect it to event since "event sponsors" rings true more than "website sponsors". Let's see what JB and/or @adarsh think.

Reply:
My thoughts exactly.

Reply:
👍🏾